res/ai/DefaultAIPlayer/TaskArchive.lua

```diff
@@ -195,4 +171,54 @@ function JobSellMovies:Tick()
 			end
 		end
 	end
+end
+
+function JobSellMovies:determineLicenceToSell(movies, toSell)
+	local licenceToSell = nil
+	local rnd = math.random(0,100)
+
+	if rnd > 90 then
```

Comment:
Hmm... Bin mir etwas unschluessig, ob `determineLicenceToSell` die 10% "mache nix" beinhalten sollte - oder ob man von der Funktion erwartet die auf alle Faelle eine Lizenz zurueckgibt (sofern die Kriterien stimmen). Das wuerde bedeuten, die random-10% muessten vorher geprueft (und nur in 90% der Faelle `determineLicenceToSell` aufgerufen) werden.

Reply:
Kann wahrscheinlich komplett raus. Die Idee war, dass auch nicht immer verkauft werden muss, wenn die Bedingungen erfüllt sind. Ich hielt das "Nichtstun" hier für OK, da ja auch sonst null zurückkommen kann (wenn bei der gewürfelten Bedingung der schlechteste Film nicht schlecht genug ist.